src/utils/platform-colors.js

```diff
@@ -62,12 +67,24 @@ export const STATIC_PLATFORMS = [
     icon: <PermanentWaterSiteIcon />,
     mapIcon: "PermanentWaterIcon",
   },
+  {
+    name: "Neutrally Buoyant Float",
+    color: "#F2B705",
+    icon: <PermanentWaterSiteIcon />,
+    mapIcon: "PermanentWaterIcon",
+  },
   {
     name: "Moored Buoy",
     color: "#8C2D04",
     icon: <MooredBuoyIcon />,
     mapIcon: "MooredBuoyIcon",
   },
+  {
+    name: "ARGO",
```

Comment:
Is this meant to have the name "ARGO" and the same icon?

Reply:
Yes, ARGO would use the `MooredBuoyIcon`, but we can add new icons if you prefer.